examples/CloudWatchEvents.py

```diff
@@ -0,0 +1,91 @@
+from troposphere import Template
+from troposphere.events import Rule, Target
+from troposphere.iam import Role, Policy
+from troposphere.awslambda import Function, Code
+from troposphere import GetAtt, Join
+
+
+t = Template()
+
+
+# Create a Lambda function that will be mapped
+code = [
+    "var response = require('cfn-response');",
+    "exports.handler = function(event, context) {",
+    "   context.succeed('foobar!');",
+    "   return 'foobar!';",
+    "};",
+]
+
+# Create a role for the lambda function
+t.add_resource(Role(
+    "LambdaExecutionRole",
+    Path="/",
+    Policies=[Policy(
+        PolicyName="root",
+        PolicyDocument={
+            "Version": "2012-10-17",
+            "Statement": [{
+                "Action": ["logs:*"],
+                "Resource": "arn:aws:logs:*:*:*",
+                "Effect": "Allow"
+            }, {
+                "Action": ["lambda:*"],
+                "Resource": "*",
+                "Effect": "Allow"
+            }]
+        })],
+    AssumeRolePolicyDocument={"Version": "2012-10-17", "Statement": [
+        {
+            "Action": ["sts:AssumeRole"],
+            "Effect": "Allow",
+            "Principal": {
+                "Service": [
+                    "lambda.amazonaws.com",
+                    "apigateway.amazonaws.com"
```

Comment:
haha thanks - i haven't contributed to this before so used your PR as a guide :)